### PR TITLE
Upgrade to Quarkus Platform 2.5.0.Final

### PR DESCRIPTION
**Pull request by zbendhiba**
Note that the `main` branch points at the latest stable Camel Quarkus release.
Pull requests should be generally send against the `camel-quarkus-main` branch pointing at the current Camel Quarkus SNAPSHOT.

### CODE REVIEW

File: aws-lambda/pom.xml [hunk 1]
```diff
@@ -21,13 +21,13 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.apache.camel.quarkus.examples</groupId>
     <artifactId>camel-quarkus-examples-aws-lambda</artifactId>
-    <version>2.4.0</version>
+    <version>2.5.0-SNAPSHOT</version>
```
Comment (ppalaga) [inline aws-lambda/pom.xml:24]: This should either be 2.5.0 or you maybe want to make as separate commit to tag 2.5.0 of the examples?

**Review by ppalaga — COMMENTED** [review]
One comment inline, otherwise looks good.

File: aws-lambda/pom.xml (hunk 1 above)
Comment (zbendhiba) [reply aws-lambda/pom.xml:24]: Yes that was you who wrote the doc 🤣
I was going to suggest changing that part. Let me update this and also the release guide.

**Comment by zbendhiba** [comment]
@ppalaga it should be fine  now

File: aws-lambda/pom.xml (hunk 1 above)
Comment (ppalaga) [reply aws-lambda/pom.xml]: Indeed, the tag commit comes after the PR in the docs. Sorry for the noise.

**Review by ppalaga — APPROVED** [review]
